### DISCUSSION TITLE
How to be able to filter on the count of a relationship ?

### DISCUSSION THREAD

**Discussion by sorasful** [collerek/ormar#590] [Q&A] [answered]
Hello there ! 

Again, thanks for this library, this is so useful to me :pray: .

I have a question, imagine you have these models 

```python
class Author(ormar.Model):
    class Meta(BaseMeta):
        tablename = "authors"

    id: int = ormar.Integer(primary_key=True)
    name: str = ormar.String(max_length=100)


class Book(ormar.Model):
    class Meta(BaseMeta):
        tablename = "books"

    id: int = ormar.Integer(primary_key=True)
    author: Optional[Author] = ormar.ForeignKey(Author)
    title: str = ormar.String(max_length=100)
    year: int = ormar.Integer(nullable=True)
```

And you would like to be able to find only the authors with at least 3 Books. 
How would you handle this ? 

I tried filtering like this : `Author.object.filter(books__count_gt=3)` , same for len, but no luck.

Thank you very much for your time !

**Comment by collerek** [comment, marked as answer]
Right now sitting on aggregates is not possible, but might be an enhancement.
